addons/im_livechat/models/res_users.py

```diff
@@ -91,6 +91,21 @@ def _compute_has_access_livechat(self):
         for user in self.sudo():
             user.has_access_livechat = user.has_group('im_livechat.im_livechat_group_user')
 
+    def write(self, vals):
+        if vals.get("group_ids"):
+            operator_group = self.env.ref("im_livechat.im_livechat_group_user")
+            if operator_group in self.all_group_ids:
```

Comment:
There is missing a condition on parsing `group_ids`.
If none of the relevant groups are in `group_ids` (depending on its format), we don't need to do anything.

Reply:
If it's easier, we might want to prevent from using some formats.
Such as only allowing commands.